README.md

```
@@ -4,7 +4,7 @@
 
 # TYPO3 extension `typo3_toolbox`
 
-[![Supported TYPO3 versions](https://badgen.net/badge/TYPO3/13/orange)]()
+[![Supported TYPO3 versions](https://badgen.net/badge/TYPO3/14/orange)]()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)